assets/style.sass

```diff
@@ -117,3 +117,12 @@ section
     color: inherit
     font-style: italic
     font-weight: $regular-weight
+
+.talk
+  margin-bottom: $grid-height
```

Comment:
This pattern of having only a bottom border hasn't been used elsewhere in the resume, but I think it  has a good effect here. An alternative would be to set the top and bottom margins to half `$grid-height`. Using the full `$grid-height` felt a little too much to me 😅